src/actions/transactions.js

```diff
@@ -79,97 +100,113 @@ export const loadTransactions = ({
  *
  * @param {String} address - address of the active account
  */
-export const loadLastTransaction = address => (dispatch, getState) => {
+export const getLastTransaction = address => async (dispatch, getState) => {
```

Comment:
The comment above says "This action is used to get the data for "My Wallet Details" module on wallet page". We don't have the last transaction in "My Wallet Details" module on wallet page anymore. IMO this action can be removed.

Reply:
thx @slaweet for the review I was checking the code and the problem is that this function is still in use in.

https://github.com/LiskHQ/lisk-hub/blob/da0946a4f0b54281e404d732cc2fb501c6bf5904/src/components/transactions/walletTransactions/walletTransactions.js#L55

And because of this I can't remove.

I checked in development and is still in use

Reply:
`getLastTransaction` is called from `walletTransactions`, but AFAIK that call can be removed as well. If you search for `getLastTransaction` in the whole project, you'll see that there is no reducer listening to the `getLastTransaction` action.

```
$ git grep getLastTransaction
src/actions/transactions.js:export const getLastTransaction = address => async (dispatch, getState) => {
src/actions/transactions.js:  dispatch(loadingStarted(actionTypes.getLastTransaction));
src/actions/transactions.js:  dispatch(loadingFinished(actionTypes.getLastTransaction));
src/components/transactions/explorerTransactions/explorerTransactions.test.js:    getLastTransaction: jest.fn(),
src/components/transactions/walletTransactions/index.js:import { getLastTransaction, getTransactions } from '../../../actions/transactions';
src/components/transactions/walletTransactions/index.js:  getLastTransaction,
src/components/transactions/walletTransactions/walletTransactions.js:    this.props.getLastTransaction(this.props.account.address);
src/components/transactions/walletTransactions/walletTransactions.test.js:    getLastTransaction: jest.fn(),
src/constants/actions.js:  getLastTransaction: 'GET_LAST_TRANSACTION',
```

Reply:
I pushed a commit to remove it https://github.com/LiskHQ/lisk-hub/pull/2312/commits/04b8f820773f2494126280fdab2fadacc612a328